src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindowCommandChangeObserver.java

```diff
@@ -4,20 +4,21 @@
 
 public class CommandManagerWindowCommandChangeObserver implements Subscriber {
 
-	private CommandManagerWindow commandManagerWindow;
+    private CommandManagerWindow commandManagerWindow;
 
-	public CommandManagerWindowCommandChangeObserver(CommandManagerWindow commandManagerWindow) {
-		super();
-		this.commandManagerWindow = commandManagerWindow;
-	}
+    public CommandManagerWindowCommandChangeObserver(CommandManagerWindow commandManagerWindow) {
+        super();
+        this.commandManagerWindow = commandManagerWindow;
+    }
 
-	public String toString() {
-		return "Current command change observer for command manager window";
-	}
+    public String toString() {
+        return "Current command change observer for command manager window";
+    }
```

Comment:
spaces instead of tabs

Reply:
Fixed in latest commit

Reply:
not really. Maybe your IDE makes spaces when saving file